nipy/labs/spatial_models/mroi.py

```diff
@@ -700,14 +700,14 @@ def select_roi(self, id_list):
         # set new features
         # (it's ok to do that after labels and id modification since we are
         # poping out the former features and use the former id indices)
-        for fid in self.features:
+        for fid in list(self.features):
             f = self.remove_feature(fid)
             sf = [f[id] for id in id_list_pos]
```

Comment:
Virgile - it looks to me as if the output from `f = self.remove_feature(fid)` is a list.  So maybe the `fid` is badly named, it should be called `feature_name` where the feature value is a list of arrays.  Is that right?